CONTENT/about.markdown

```diff
@@ -8,42 +8,49 @@
 Our mission is to empower kids with programming skills for making social change!
 
 With wizard amigos workshops
-Kids will:
-- ...
-Tech Companies will:
-- ...
-Mentors will:
-- ...
+Kids will be able to:
+- get to know basic programming concepts and develop logical thinking
+- learn fastest growing programming language - JavaScript and its ecosystem
+- learn to collaborate with other learners
+- find out everything about how to apply her/his new skills with learning-by-doing approach
+
+Tech Companies will be able to:
+- identify and access market opportunities for educational applications and classroom integration of new products.
+
+Mentors will be able to:
+- continue learning 
+- get paid for sharing their skills with kids
+- be part of community of continuous learners and get help with getting their projects off the ground
+
 ## Our story
-Jan 2014 we started Coding Amigos Hackathons, we organized over 70 events and co-learning workshops. Before that we participated in several groups- from universities, incubators, rails girls, startup weekends, open tech school, ...
+In early 2014 we started Coding Amigos Hackathons in Berlin. We organized over 70 events and co-learning workshops. Before that we participated in several groups: University Incubator, Social Media Week, Rails Girls, StartUp weekend, Open Tech School etc.
 
-During last year,we have met a lot of coding learners and got to know several techniques for beginners and advanced learners.
+During the last year we have met a lot of coding learners and got to know several techniques for beginners and advanced learners. We realised we have enough experiences to bring this skills also to the youngest segment. And so the idea for programming school for kids was born.
```

Comment:
+1, but:
I would like to not use the terms:
`school`, `teacher`, `teaching` so much. Because I believe what we do is not a school, nor a university or academy. I am kind of ok with the word `institute` - wikipedia says:
> An institute is an organisational body created for a certain purpose.
and thats kind of true :-)

We `mentor` kids. We `learn together` - `teachers` are `learners` themselves.
Once kids collaborate, they `teach each other`, so they `learn together` or they just `collaborate` and learning is a side effect. We also `inspire` them. ... I would like to avoid the traditional wording present in the mainstream education system in order to give more rooms for different perception of what a `learning process` could look like and also to give us and others more space and freedom to brainstorm about it :-)

Once punch line nodeschool is using is `learn you node` instead of `teach you node` or `teach yourself node` :-)
What do you think about that?
We already have `magic` and `sessions` and thought of `jams` and maybe we can use lots of differently named activities...